helpers/import.js

```diff
@@ -0,0 +1,15 @@
+function importHelper(type, file) {
+  switch (process.env.moduleFormat) {
```

Comment:
Can we add a line to the JS README mentioning this option (and maybe an example of it in use)?

Reply:
Good point - I've added information to the README about how to list the generator options. In this case I think the effect should be relatively obvious, so have not included an example.